src/book.js

```diff
@@ -577,20 +608,24 @@ export class Book {
    * PDF builder base function for Classic (non-Wacky) layouts. Called by [createoutputfiles]
    *
    * @param {Object} config
+   * @param {number} config.maxSigCount
    * @param {PageInfo[][]} config.pageIndexDetails : a nested list of objects.
    */
   async createSignatures(config) {
+    console.log("createSignatures ",config)
```

Comment:
this method should be renamed (and the call chain audited for similar renaming) -- was lead to false assumptions -- this is `createSignature` singular. The breaking apart happens up in `createoutputfiles` which was not obvious/clear